### DISCUSSION TITLE
Remove connections from the DevTools

### DISCUSSION THREAD

**Discussion by SatkunamSuganthar** [Power-Maverick/DataverseDevTools-VSCode#39] [Q&A] [answered]
Hi All, @Power-Maverick Thanks for this wonderful tool. While I was attending the 4 days Course conducted by Danish, I have come across an issue to remove the connection I created

Is there any way to remove the connections which were created in the past in the Dataverse DevTools UI. 

@Power-Maverick Could you please confirm that this feature (Delete connections) implemented this or in the features list to be added?

**Comment by Power-Maverick** [comment]
If I understood you correctly; you would like to remove a connection. For that, you can simply right-click on the connection and click on **Delete Connection** option. Lemme know if that answered your question.

![image](https://user-images.githubusercontent.com/36135520/148068886-b22c4a90-6c4e-41f0-8649-c7be66b5815c.png)

**Comment by SatkunamSuganthar** [comment]
Thanks for the prompt reply @Power-Maverick and I saw the method of the delete  connection in the one of the helper class in the Repo.  Unfortunately, I will not have any of the options available even to Connect to Dataverse as it is shown in the screenshot. when I right click, nothing happens. 

The version of the Dev Tool is 1.1.10

The First Authentication option I have tried is Microsoft login Prompt. I wanted to connect using the Client Secret Auth Type before I need to delete the connection which is shown below if possible.

My screen shot looks like as below

![image](https://user-images.githubusercontent.com/97066265/148127945-e1f58b15-b647-4798-9d9e-d9d665e2746d.png)

Is there any way I can manually delete the connections in the config file which is in the extension folder.

**Comment by Power-Maverick** [comment]
Thanks for bringing this up to my attention. It seems this is an issue and @thomassandsor helped me replicate the issue this morning. I have resolved it but I need to release that version yet. I will update here once it is released.

**Reply by SatkunamSuganthar** [reply]
Thanks @Power-Maverick. Will it possible to get the early access for the latest release if the changes has been committed to remote repo ?

**Reply by Power-Maverick** [reply]
See below. If it works please mark the below as answered.

**Reply by SatkunamSuganthar** [reply]
Yes it worked as I was able to delete the all connections and recreate them. Thanks @Power-Maverick

**Comment by Power-Maverick** [comment, marked as answer]
This issue should be fixed in v1.1.11
Make sure to first use the command "Delete All Connections" that will clean the connections; then re-create the connections and make sure not to specify `Dev`, `Test`, `QA`, `Prod` in the friendly name; those are reserved words.
cc: @SatkunamSuganthar & @thomassandsor

**Reply by thomassandsor** [reply]
Stupid question, but where do I enter that command?  :)

**Reply by Power-Maverick** [reply]
1. `Ctrl` + `Shift` + `P` to open the command palette
2. Search for "Dataverse DevTools: Delete All Connections"
3. On confirmation box; click `OK`

This should clear/delete all connections; including faulty/erroneous
